libr/io/io.c

```diff
@@ -1,37 +1,25 @@
-/* radare - LGPL - Copyright 2008-2017 - pancake */
+/* radare2 - LGPL - Copyright 2008-2017 - condret, pancake, alvaro_fe */
 
-#include "r_io.h"
-#include "r_util.h"
-#include <stdio.h>
-
-#ifdef _MSC_VER
-#pragma comment(lib, "advapi32.lib")
-#endif
+#include <r_io.h>
+#include <sdb.h>
+#include "../config.h"
```

Comment:
Just use `#include <config.h>` using relative paths is bad imho (plus it breaks appveyor build)